src/lib/storage/index/b_tree/b_tree_index.cpp

```diff
@@ -0,0 +1,32 @@
+#include "b_tree_index.hpp"
+
+#include "storage/index/column_index_type.hpp"
+#include "resolve_type.hpp"
+
+namespace opossum {
+
+  BTreeIndex::BTreeIndex(const std::vector<std::shared_ptr<const BaseColumn>> index_columns)
+      : BaseIndex{get_index_type_of<BTreeIndex>()}, _index_column(index_columns[0]) {
+    DebugAssert((index_columns.size() == 1), "BTreeIndex only works with a single column.");
```

Comment:
Let's make this a non-debug assert.

Reply:
Done and also changed this for the other indices.